frontend/src/metabase-types/api/writeback-form-settings.ts

```diff
@@ -0,0 +1,45 @@
+export type FormType = "inline" | "modal";
+export type FieldType = "text" | "number" | "date" | "category";
+export type InputType =
+  | "text"
+  | "longtext"
+  | "number"
+  | "date"
+  | "datetime"
+  | "monthyear"
+  | "quarteryear"
+  | "dropdown"
+  | "inline-select";
+export type Size = "sm" | "md" | "lg";
```

Comment:
nit: maybe use full names like `small`, `medium`, and `large`?
Short names make me thing about CSS breakpoints